tutorials/dockerized-versions.md

```diff
@@ -0,0 +1,145 @@
+# Dockerized versions
+[[toc]]
+
+::: warning
+To use the dockerized versions of Pterodactyl you need to install Docker!
+For a quick start you can run `curl -sSL https://get.docker.com/ | CHANNEL=stable bash`. For more information refer to the official [Docker documentation](https://docs.docker.com/engine/install/).
+:::
+
+## Panel Image
+`ghcr.io/pterodactyl/panel` is a ready to use docker image for the panel.
+
+### Requirements
+The docker image requires some additional software to function. The software can either be provided in other containers (see the [docker-compose.yml](https://github.com/pterodactyl/panel/blob/1.0-develop/docker-compose.example.yml) as an example) or as existing instances.
+
+A mysql database is required. We recommend the stock [MariaDB Image](https://hub.docker.com/_/mariadb/) image if you prefer to run it in a docker container. As a non-containerized option we recommend mariadb.
+
+A caching software is required as well. We recommend the stock [Redis Image](https://hub.docker.com/_/redis/) image. You can choose any of the [supported options](#cache-drivers).
+
+You can provide additional settings using a custom `.env` file or by setting the appropriate environment variables in the docker-compose file.
+If using the docker-compose file the location of a custom `.env` file would be `/srv/pterodactyl/var/.env`. 
+
+### Setup
+Start the docker container and the required dependencies (either provide existing ones or start containers as well, see the [docker-compose.yml](https://github.com/pterodactyl/panel/blob/1.0-develop/docker-compose.example.yml) file as an example).
+```bash
+# If you are using docker compose
+wget -O docker-compose.yml https://github.com/pterodactyl/panel/blob/1.0-develop/docker-compose.example.yml
```

Comment:
Should this not use raw? Else it downloads html?

Reply:
Yes, thanks for pointing out!